pyiceberg/catalog/glue.py

```diff
@@ -109,6 +109,10 @@
 GLUE_SKIP_ARCHIVE = "glue.skip-archive"
 GLUE_SKIP_ARCHIVE_DEFAULT = True
 
+# Configure an alternative endpoint of the Glue service for GlueCatalog to access.
+# This could be used to use GlueCatalog with any glue-compatible metastore service that has a different endpoint
+GLUE_CATALOG_ENDPOINT = "glue.endpoint"
```

Comment:
Same as in Java 👍 
 https://github.com/apache/iceberg/blob/d9dbb75ed057c4f6673626763f257c0a9049e249/aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java#L103